internal/api/web/views/components/dashboard_components/dashboard_home_page/quiz_tile.templ

```diff
@@ -0,0 +1,25 @@
+package dashboard_components
+
+import "github.com/Molnes/Nyhetsjeger/internal/data/quizzes"
+
+templ QuizTile(quiz quizzes.Quiz) {
+	<a
+		class="mr-4 mb-4 bg-gray-200 shadow-md rounded-lg max-w-56 overflow-hidden flex flex-col hover:shadow-lg transition duration-100 ease-in-out transform hover:-translate-y-1 hover:scale-105"
+		href={ templ.SafeURL("/dashboard/quiz/edit/" + quiz.ID.String()) }
```

Comment:
Update to use /edit-quiz with query params

Reply:
72dddb8034a6e4362dfb908d36e500d093230131